tests/unit/plugins/modules/test_lambda_event.py

```diff
@@ -375,3 +384,143 @@ def test_lambda_event_stream_update_event(
         api_params.update({"FunctionName": function_name, "UUID": existing_event_source[0]["UUID"]})
         assert dict(changed=True, events=event_source_updated) == result
         client.update_event_source_mapping.assert_called_once_with(**api_params)
+
+
+@pytest.mark.parametrize(
+    "params, expected, exception, message, source_type",
+    [
+        (
+            {
+                "source_arn": "arn:aws:sqs:us-east-1:123456789012:ansible-test-28277052.fifo",
```

Comment:
We should probably also include a unit test for the `sqs` source type that does not include passed `batch_size` and `maximum_batching_window_in_seconds` to ensure default values are set as expected.

Reply:
Done!